cli/cmd/vuln_host.go

```diff
@@ -360,6 +387,18 @@ func init() {
 		vulHostShowAssessmentCmd.Flags(),
 	)
 
+	setFailOnSeverityFlag(
+		vulHostListCvesCmd.Flags(),
```

Comment:
why are we adding these flags to the sub-command `list-cves`? 🤔

Reply:
No, originally planned to add that here but decided against it. Will remove that